index.html

```diff
@@ -46,7 +46,7 @@
 				<h2>Country</h2>		
 				<select name="country">
 					<option value selected diasabled>Choose your country</option>
-				  	<option value="AF">Afghanistan</option>
+				  <option value="AF">Afghanistan</option>
```

Comment:
In sublime, go to the white space that precedes the `option` tag that is out of place and see if it is a different kind of white space that the other lines. It might be spaces, but the other lines are using tabs, visa versa, etc.